packages/rspack/src/util/index.ts

```diff
@@ -106,3 +108,12 @@ export function toJsAssetInfo(info?: AssetInfo): JsAssetInfo {
 		...info
 	};
 }
+
+const yellow = (content: string) =>
+	`\u001b[1m\u001b[33m${content}\u001b[39m\u001b[22m`;
+export const deprecatedWarn = (content: string, enable = true) => {
+	if (enable) {
+		console.warn(yellow(content));
+	}
+};
+export const termlink = terminalLink;
```

Comment:
no seen any usage in code

Reply:
Will use it to add some link in deprecate message in the following PR